assets/js/text_container.js

```diff
@@ -0,0 +1,26 @@
+    var collapse_r = false
+
+    $(document).ready(function () {
+        $('#sidebarCollapse_l').on('click', function () {
+            $('#sidebar_l').toggleClass('active');
+            var el_l = document.getElementById('sidebarCollapse_l');
+            if ($('#sidebar_l').css("marginLeft")=='0px'){
+                el_l.firstChild.data = "<";
+            }else{
+                el_l.firstChild.data = ">";
+            }
+        });
+    });
+
+    $(document).ready(function () {
+        $('#sidebarCollapse_r').on('click', function () {
+            $('#sidebar_r').toggleClass('active');
+            var el_r = document.getElementById('sidebarCollapse_l');
+
+            if (el_r.firstChild.data = "<"){
+                el_r.firstChild.data = ">";
+            }else{
+                el_r.firstChild.data = "<";
+            }
```

Comment:
If I am reading this right, this means that when you click on `sidebarCollapse_r`, it checks the `.firstChild.data` of `sidebarCollapse_l`and changes its data instead of changing the data of `sidebarCollapse_r`. Is that correct? I haven't loaded it up and tested it, but that is what it looks like to me.